proto/pdpb.proto

```diff
@@ -1,303 +1,321 @@
-syntax = "proto2";
+syntax = "proto3";
```

Comment:
I think it is ok, they are wire compatible.  Beside compatibility, is there any thing you are concerned about?